### DISCUSSION TITLE
`position` property in `Location` object

### DISCUSSION THREAD

**Discussion by 838** [readium/architecture#168] [Q&A] [answered]
Could the `position` property hold an index value of the `readingOrder`?

https://readium.org/architecture/models/locators

**Comment by mickael-menu** [comment, marked as answer]
Yes, that's basically what is done for comic books. But you need to start the `position` index at 1 instead of 0. The reason is that this is used for display purposes, similar to page numbers.

**Reply by 838** [reply]
@mickael-menu Thank you 🙏

**Reply by jerome65536** [reply]
Not sure whether an integer chapter index is suitable for chapter numbers for display purposes in general — what about chapter numbers such as _Preface_, _III_, _II_, _I_, _Table of contents_, _1_, _2_, _3_, _Appendix A_.

**Reply by mickael-menu** [reply]
Quoting the [position list specification](https://readium.org/architecture/models/locators/positions/):

> While reading a publication, users may need to reference or access a specific position.
>
> While some formats have an implicit (image based publications) or an optional explicit option (EPUB 3.x) to express such references, clients need references even for publication formats that do not provide anything.

You can view the `position` property as something similar to a physical page number, it's not a "chapter number".

So in the case of comic books, it makes sense to have one position per reading order item. For reflowable formats like EPUB, you will likely have multiple positions per resource. [See this issue for some implementation guidelines](https://github.com/readium/architecture/issues/101).
